### PR TITLE
prep-javascript-arrays - Lamberto Martinez

### PR DESCRIPTION
**Pull request by Lambpato**
Screen Shot 2022-09-27 at 1 32 36 PM

### CODE REVIEW

File: prep-javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+var thirdElement = languages[2];
+
+console.log('The 3rd elemtn of the array is', thirdElement);
+
+var length = languages.length;
+
+console.log('The length of the array is', length);
+
+var lastElement = languages[5];
```
Comment (thebearingedge) [inline prep-javascript-arrays/main.js:17]: We can get the last element of any array by subtracting 1 from its length.

```js
var lastElement = languages[length - 1];
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-javascript-arrays/main.js (hunk 1 above)
Comment (Lambpato) [reply prep-javascript-arrays/main.js:17]: ahh ok I was going of prior knowledge my bad

**Comment by Lambpato** [comment]
Screen Shot 2022-09-27 at 5 42 37 PM

File: prep-javascript-arrays/main.js (hunk 1 above)
Comment (Lambpato) [reply prep-javascript-arrays/main.js]: Ive corrected it

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**